mlflow/models/evaluation/base.py

```diff
@@ -98,8 +98,9 @@ def eval_fn(
                     Refer to the DefaultEvaluator behavior section for what metrics
                     will be returned based on the type of model (i.e. classifier or regressor).
                 :param kwargs: Includes a list of args that are used to compute the metric. These
-                    args could information coming from input data, model outputs or parameters
-                    specified in the `evaluator_config` argument of the `mlflow.evaluate` API.
+                    args could be information coming from input data, model outputs, other metrics,
```

Comment:
@annzhang-db Can we add an example to our evaluation docs to demonstrate this? Docstrings are great, but we need a guide :)